### ISSUE TITLE
Remove newlines

### ISSUE THREAD

**Issue by evilebottnawi** [prettier/plugin-php#27]
Input:
```php
if (version_compare(PHP_VERSION, '7.1.0', '<')) {
    insteria_error('Minimum required PHP version is 7.1.0', 'Insteria Theme Requirement Checker');
}

$autoloaderPath = __DIR__ . '/vendor/autoload.php';
```

Output:
```php
if (version_compare(PHP_VERSION, '7.1.0', '<')) {
    insteria_error('Minimum required PHP version is 7.1.0', 'Insteria Theme Requirement Checker');
}
$autoloaderPath = __DIR__ . '/vendor/autoload.php';
```

I think we should newlines as is, it is compatibility with PSR.
Also i'm interested in helping, right now i'm support `css/scss/sass` in `prettier` (https://github.com/prettier/prettier/graphs/contributors). You can invite me and i will help to improve output :star:

**Comment by mgrip** [comment]
Thanks @evilebottnawi! I just checked and this seemed to run correctly for me - are you sure you have the latest changes? Line spacing should have been fixed with https://github.com/prettier/prettier-php/commit/515346975c84be52ca93944764ced2f0a322d6e7 and https://github.com/prettier/prettier-php/commit/d4ec7228f4027c8d1f6db75bc3e15299b13027d8

And we'll take all the help we can get, feel free to assign yourself to an issue or put up a PR!
